### DISCUSSION TITLE
what is the difference of lightweight build, not clear

### DISCUSSION THREAD

**Discussion by badiku** [jart/cosmopolitan#688] [General]
there are many different version to download at https://redbean.dev/

both PE+ELF+MachO+ZIP+SH, but what is the difference, not clear

> redbean-2.2.com
> 2.3m - PE+ELF+MachO+ZIP+SH

and 
> redbean-tiny-2.2.com (lightweight build)
> 1.6m - PE+ELF+MachO+ZIP+SH

**Comment by pkulchenko** [comment]
My understanding is that there are too many (small) differences to list. You can check yourself by finding all `IsTiny` references in redbean.c (they will [look like this one](https://github.com/jart/cosmopolitan/blob/master/tool/net/redbean.c#L141))
